tests/scripts/pre-validation.sh

```diff
@@ -10,6 +10,7 @@ DA_DIR="solutions/banking"
 TERRAFORM_SOURCE_DIR="tests/resources/existing-resources"
 JSON_FILE="${DA_DIR}/catalogValidationValues.json"
 REGION="us-south"
+SECRETS_MANAGER_GUID=$(curl -X GET --retry 3 -fLsS https://raw.githubusercontent.com/terraform-ibm-modules/common-dev-assets/main/common-go-assets/common-permanent-resources.yaml | yq e '.secretsManagerGuid')
```

Comment:
no need to do this either - common-dev-assets will be locally available in the git submodule - so just pull it locally